views/layouts/main.handlebars

```diff
@@ -59,7 +59,7 @@
 	<div class="sponsors">
     <a href="http://liaison.com/" target="_blank"><img src="/static//img/liason.png"></a>
     <a href="http://siu.edu/" target="_blank"><img src="/static/img/siu.png"></a>
-    <a href="http://www.harnessteched.com/" target="_blank"><img src="/static/img/teched.jpg"></a>
+    <a href="http://www.harnessteched.com/" target="_blank"><img src="/static/img/harnessteched.png"></a>
```

Comment:
Can we clean this up so that it links to `http://www.harnessdigitalmarketing.com/` and we name the image `harnessdm` or something?